src/Extensions/Hosting.Services.Web/KestrelListenerBuilder.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+using System.IO;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.ServiceFabric.Services.Communication.AspNetCore;
+using Microsoft.ServiceFabric.Services.Communication.Runtime;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Web
+{
+	/// <summary>
+	/// Creates ServiceInstanceListener with all of Omex dependencies initialized
+	/// </summary>
+	internal sealed class KestrelListenerBuilder<TStartup, TServiceContext> : IListenerBuilder<TServiceContext>
```

Comment:
It's an internal class and the only public method of it described in the interface. Everything else could be understood from code just looking at class. Please point out unclear parts so maybe it could be solved by renaming methods and comments without 
condescending to comments :) I'm okay with adding comments, but only if there are no other way